apps/test/unit/templates/SchoolAutocompleteDropdownTest.js

```diff
@@ -1,78 +1,90 @@
 import React from 'react';
```

Comment:
The `SchoolAutocompleteDropdown` component uses [the Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) to send search queries to the server.  Fetch isn't stubbed by sinon's fake server, [by design](https://github.com/sinonjs/nise/issues/7).  This was passing in Phantom because Phantom doesn't implement fetch, and falls back on a polyfill that uses XMLHttpRequest, which _is_ stubbed by sinon's fake server. But running in Chrome we use real Fetch, so the fake server doesn't work.

The solution here is to stub fetch directly instead of a using a fake server. (This also works for Phantom because we're stubbing the polyfill instead of part of its underlying implementation.)  That's a pretty significant change but I think we ended up in a good place.  Also cleaned up some tests here to use async/await which allows them to read in order.